GeneratorAnalysis/Scripts/Constants.h

```diff
@@ -56,6 +56,35 @@ namespace Constants {
     };
     static const TString LabelXAxisTwoDCosOpeningMuonTotalProtonInMuonCosTheta = ";cos(#theta_{#vec{p}_{#mu},#vec{p}_{sum}}) [bin #]";
 
+    // Variables for double differential analysis - GKI case
+    static const int TwoDNBinsMuonCosThetaThreeD = 2; 
+    std::vector<double> TwoDArrayNBinsMuonCosThetaThreeD{-1.0,0.5,1.0};
```

Comment:
Is there a difference between this variables and `TwoDArrayNBinsMuonCosTheta`/`TwoDNBinsMuonCosTheta` defined above?